cdap-ui/app/features/dashboard/controllers/widget.js

```diff
@@ -136,7 +162,6 @@ angular.module(PKG.name+'.feature.dashboard')
     });
     $scope.wdgt.fetchData($scope);
     $scope.chartHistory = null;
-    $scope.stream = null;
```

Comment:
Ah ok. I'll undo this.
I thought by default its 'undefined', didn't see the point of setting it to null.